SportsHub/UnitTests/Controllers/ArticleControllerTests.cs

```diff
@@ -1,134 +1,108 @@
-﻿using AutoMapper;
-using FluentValidation;
-using FluentValidation.TestHelper;
+﻿using AutoFixture;
+using AutoFixture.AutoMoq;
+using AutoFixture.Xunit2;
+using AutoMapper;
 using Microsoft.AspNetCore.Mvc;
 using Moq;
 using SportsHub.Api.Controllers;
 using SportsHub.Api.Mapping.Models;
-using SportsHub.Api.Validations;
-using SportsHub.AppService.Authentication.Models.DTOs;
 using SportsHub.AppService.Services;
 using SportsHub.Domain.Models;
-using UnitTests.MockData;
 using Xunit;
 
 namespace UnitTests.Controllers
 {
     public class ArticleControllerTests
     {
         private readonly Mock<IArticleService> _articleService;
-        private readonly Mock<IGenerateModelStateDictionary> _generateModelStateDictionary;
         private readonly ArticleController _articleController;
-        private readonly ArticleValidation _articleValidation;
         private readonly IMapper _mapper;
-
+        private IFixture _fixture;
+    
         public ArticleControllerTests()
         {
+            SetupFixture();
             if (_mapper == null)
             {
                 var mappingConfig = new MapperConfiguration(x => x.CreateMap<Article, ArticleResponseDTO>());
```

Comment:
In both Controllers is like this maybe we can be changed when we create the task to refactor more of the tests because beside this UserServiceTests need another thests for added method and ArticleControllerTests need new dependancy to work